scripts/publicNames.txt

```diff
@@ -1214,6 +1214,13 @@ plugins
                         getToken
                         getUrl
                         getUser
+                cilogon
```

Comment:
These lines just need to be moved up so that cilogon is alphabetical.  The needed diff is attached 
[diff.txt](https://github.com/user-attachments/files/18457495/diff.txt)